torch/functional.py

```diff
@@ -1217,7 +1217,8 @@ def norm(input, p="fro", dim=None, keepdim=False, out=None, dtype=None):  # noqa
         Use :func:`torch.linalg.norm` instead.
 
     Args:
-        input (Tensor): the input tensor
+        input (Tensor): the input tensor. If dim is None and x has more than 2 dimensions,
```

Comment:
The description of what happens when dim is none seems more appropriate for the "dim" entry, not the "input" entry.

Reply:
Yes, my initial thought. Tried to follow the `torch.linalg.norm` docs to have more unified docs in the end. Thought since it's new, it was reviewed thoroughly. Maybe this should be fixed there too (moved to the "dim" entry).